### PR TITLE
Remove --partition_config_path and --generate_sample_info_table flags.

### PR DESCRIPTION
**Pull request by tneymanov**
Should be submitted after #553

### CODE REVIEW

**Comment by samanvp** [comment]
Sorry, I think because of my last a few PRs you need to update this PR:
* #550 removed `--partition_config_path`.
* #536 modified where we use `--generate_sample_info_table` (but still this flag needs to be deleted).

Thanks!

**Comment by tneymanov** [comment]
Done.

File: gcp_variant_transforms/options/variant_transform_options.py [hunk 1]
```diff
@@ -136,25 +136,16 @@ def add_arguments(self, parser):
...
-              'sample_id can be used to distinguish between sample names. '
-              '[EXPERIMENTAL]'
-             ).format(sample_info_table_schema_generator.TABLE_SUFFIX))
+        default='gcp_variant_transforms/data/sharding_configs/'
+                'homo_sapiens_default.yaml',
+        help=('File containing list of shards and output table names. You '
+              'can use provided default sharding_config file to split output '
+              'by chromosome (one table per chromosome) which is located at: '
+              'gcp_variant_transforms/data/sharding_configs/'
+              'homo_sapiens_default.yaml'))
```
Comment (samanvp) [inline gcp_variant_transforms/options/variant_transform_options.py]: Any specific reason you decided to update the doc we agreed on in the previous PR?

File: gcp_variant_transforms/testing/data/sharding_configs/README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 This file summarizes the contents and the purpose for each files/folder within
 current folder.
 
-All files in this folder are used as inputs for --partition_config_path flag.
```
Comment (samanvp) [inline gcp_variant_transforms/testing/data/sharding_configs/README.md]: Again, thanks for fixing this.

File: gcp_variant_transforms/options/variant_transform_options.py (hunk 1 above)
Comment (tneymanov) [reply gcp_variant_transforms/options/variant_transform_options.py]: Nice catch! Sync error, applied your changes myself (that's how I caught the other issues I guess) - reverted.

File: docs/large_inputs.md [hunk 3]
```diff
@@ -15,7 +15,8 @@ Default settings:
   --disk_size_gb <default 250> \
   --worker_disk_type <default PD> \
   --num_bigquery_write_shards <default 1> \
-  --partition_config_path <default None> \
```
[the remark this answers is not in the document]
Comment (tneymanov) [reply docs/large_inputs.md]: Thanks to the sync issues :)

File: gcp_variant_transforms/testing/data/sharding_configs/README.md (hunk 2 above)
Comment (tneymanov) [reply gcp_variant_transforms/testing/data/sharding_configs/README.md]: Sure :)

**Pull request merged**